Add test image dockerfile

Maunty opened the pull request:
In order to perform CI checks docker file is added along with codebase.

Note: build context should be repo root.

@Stolb27 do not merge plz this PR until we finish all tasks. It`s statuses will be acceptance criteria off all work.

Stolb27 commented on arenadata/Dockerfile:
```diff
@@ -0,0 +1,42 @@
+ARG sigar=https://ci.arenadata.io/artifactory/ADB/6.7.1_arenadata4/centos/7/community/x86_64/sigar-1.6.5-163.el7.x86_64.rpm
+ARG sigar_headers=https://ci.arenadata.io/artifactory/ADB/6.7.1_arenadata4/centos/7/community/x86_64/sigar-headers-1.6.5-163.el7.x86_64.rpm
+
+RUN rpm -i $sigar && rpm -i $sigar_headers && mkdir /home/gpadmin/bin_gpdb
```
Move WORKDIR that creates `/home/gpadmin` higher this line or add mkdir option -p.

Maunty committed:
Add test image dockerfile

In order to perform CI checks docker file is added along with codebase.
Note: build context should be repo root.

Maunty commented:
@Stolb27 size lowered.
![Screenshot from 2021-03-31 17-48-00](https://user-images.githubusercontent.com/26063803/113163984-5b7b3080-9249-11eb-83c2-5df042b5a390.png)
